src/nft/components/collection/CollectionStats.tsx

```diff
@@ -299,13 +303,19 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
               {floorPriceStr} ETH
             </StatsItem>
           ) : null}
-          {stats.stats?.one_day_floor_change ? (
-            <StatsItem label="24-Hour Floor" shouldHide={false}>
-              <PercentChange>
-                {floorChangeStr}% {arrow}
+          {stats.stats?.one_day_floor_change !== undefined ? (
```

Comment:
can you actually make a variable for this at the top of the component? it's re-used a few times